tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -19,13 +19,23 @@ Create a draft release
     user create test release via api    ${PUBLICATION_ID}    AY    2021
 
 Approve a methodology for publishing immediately
-    user creates approved methodology for publication    ${PUBLICATION_NAME}
+    user creates methodology for publication    ${PUBLICATION_NAME}
+
+    user clicks link    Manage content
+    user creates new content section    1    Methodology content section 1    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+    user adds text block to editable accordion section    Methodology content section 1
+    ...    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+    user adds content to accordion section text block    Methodology content section 1    1
+    ...    Adding Methodology content    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+
+    user approves methodology for publication    ${PUBLICATION_NAME}
 
 Verify that the publication is not visible on the public methodologies page without a published release
-    user navigates to methodologies page
+    user navigates to public methodologies page
     user checks page does not contain    ${PUBLICATION_NAME}
 
 Verify that the methodology is not publicly accessible by URL without a published release
+    [Tags]    NotAgainstLocal
```

Comment:
Just wondering why you've added the `notAgainstLocal` tag here? Are you running the public frontend in production mode?

Reply:
I haven't been. I'll remove these and start running the tests locally against the frontend in production mode.